### DISCUSSION TITLE
history-graph: reporting "no history" when entity was not changed

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#17479] [Dashboard] [answered]
Consider a scenario - some entity has a constant value (or was not changed) within some period after creation.
Or imagine that a DB was replaced by a new one (deleted by a user or replaced automatically after a failure).

What we can see in a History for these entities (`input_boolean`, `input_number`, template `binary_sensor`):

![image](https://github.com/home-assistant/frontend/assets/71872483/036cd84d-5042-4dd6-a11b-729fdbbe8bae)

Only after changing these entities their History is shown:

![image](https://github.com/home-assistant/frontend/assets/71872483/9f4cb08b-b08b-4f22-a1f9-10929e9d3405)

I wonder why it is not possible to show a constant value on History?
Let's consider entities INCLUDED in Recorder, hence supposedly stored in DB.

**Comment by ildar170975** [comment]
Same here: https://github.com/home-assistant/frontend/issues/19139

**Comment by karwosts** [comment, marked as answer]
Fixed in 2024.3
